src/web/timeline.rkt

```diff
@@ -178,6 +179,28 @@
 (define (average . values)
   (/ (apply + values) (length values)))
 
+(define (render-phase-mixed-sampling mixsample)
+  `((dt "Precisions")
+    (dd (details
+         (summary "Click to see histograms")
+         ,@(for/list ([rec (in-list (group-by first mixsample))])
```

Comment:
BTW, it would be cool to sort these by total time for that operation, so that the most important operations were at the top. Not sure how easy that would be, but you could make this `for/list` output the HTML and also the total time, and then sort by total time before actually including the HTML in the page.

Reply:
Will try to implement it